great-idea/css/index.css

```diff
@@ -62,3 +62,86 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+/*My code starts here*/
+
+/*Header*/
+
+header{
+	height: 40px;
+}
+
+header a {
+	display: inline-block;
+	color: gray;
+	text-decoration: none;
+	margin: 40px 70px;
+}
+
+.logo{
```

Comment:
Position absolute is betraying you right now, use the width of the logo to fill the header container and you'll have a good day. :+1: